go.mod

```diff
@@ -3,51 +3,99 @@ module github.com/mattermost/mattermost-plugin-starter-template
 go 1.19
```

Comment:
A bit old of a Go version. Should we bump to 1.20?

Reply:
Sure, makes sense.